config.yml

```diff
@@ -25,6 +27,11 @@ defaults:
     # taskcluster.fromNow, notice this should be negative!
     taskExpirationDelay:          '- 30 minutes'
 
+    # Time extension of the expires property for task-group definitions.
+    # This is properties holding the schedulerId for a task-group, hence, we
+    # expire them later to reduce risk of abuse
```

Comment:
Create a task with different schedulerId in a task-group that already exists... even if the tasks in the group and the group definition is racing to expire.